sdk/appconfiguration/tests.yml

```diff
@@ -8,9 +8,22 @@ stages:
         - name: azure-data-appconfiguration
           groupId: com.azure
           safeName: azuredataappconfiguration
+      TimeoutInMinutes: 90
       EnvVars:
         AZURE_TEST_MODE: RECORD
         AZURE_APPCONFIG_CONNECTION_STRING: $(AZURE_APPCONFIG_CONNECTION_STRING)
         AZURE_CLIENT_ID: $(aad-azure-sdk-test-client-id)
         AZURE_CLIENT_SECRET: $(aad-azure-sdk-test-client-secret)
         AZURE_TENANT_ID: $(aad-azure-sdk-test-tenant-id)
+      PostSteps:
+        - script: |
+              python --version
+              python -m pip install setuptools termcolor in_place
+              python sdk/spring/scripts/replace_util.py --module appconfiguration --log debug --color false
+          displayName: 'Run replace_util.py'
+        - task: Maven@3
+          displayName: 'Run tests again.'
+          inputs:
+              mavenPomFile: pom.xml
+              goals: clean verify
+              options: '-Dgpg.skip -Dmaven.wagon.http.pool=false -Dmaven.javadoc.skip=true -Drevapi.skip=true -pl $(ProjectList)'
```

Comment:
We need to do more things: 
1. Update spring readme. Refs: https://github.com/Azure/azure-sdk-for-java/pull/18916/files#diff-c7e14eceb0616ce76f6f62363e88d185d7a7a7034b17698aacfb4875a2aa1bddL61
2. create 3 new samples. Refs: https://github.com/Azure/azure-sdk-for-java/pull/18916/files#diff-f47d3e56cf8f644af240ca6558498ccabb26af07fe19fb38b31c1d60aaba0ed7L1

Reply:
done